firestore/FirestoreSwiftUIExample/FirestoreSwiftUIExampleApp.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  FirestoreSwiftUIExampleApp.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+import Firebase
+
+@main
+struct FirestoreSwiftUIExampleApp: App {
+
+    init() {
```

Comment:
Not sure about the QuickStarts, but Firebase iOS SDK has an official indent of 2 spaces. They also have a nice formatter script to save you the hassle of formatting the code yourself :-)

Reply:
Thanks! I was going to ask Morgan about this today :)